### PR TITLE
Remove plus sign (+) in URL query string while per_page switching

### PR DESCRIPTION
**Pull request by Fivell**

### CODE REVIEW

File: app/assets/javascripts/active_admin/lib/per_page.js.coffee [hunk 1]
```diff
@@ -18,8 +18,12 @@ class ActiveAdmin.PerPage
     params = {}
     re = /([^&=]+)=([^&]*)/g
     while m = re.exec(query)
-      params[decodeURIComponent(m[1])] = decodeURIComponent(m[2])
+      params[@_decode(m[1])] = @_decode(m[2])
     params
+   
+  _decode: (value) ->
+  #replace "+" before decodeURIComponent
```
Comment (timoschilling) [inline app/assets/javascripts/active_admin/lib/per_page.js.coffee]: Comment need's more indent

**Comment by timoschilling** [comment]
This is a fix for #3955?

**Comment by Fivell** [comment]
@timoschilling , yes it is.
I will add fixes for @houndci  a bit later.

**Comment by Fivell** [comment]
@timoschilling , done

**Pull request merged**